make_apkovl.sh

```diff
@@ -3,6 +3,6 @@
 # Disables adding resource-forks on MacOS
 export COPYFILE_DISABLE=true
 
-cp README.md home/lncm/README.md
+cp README.md home/lncm/README.txt
 
-tar czf box.apkovl.tar.gz --exclude '.DS_Store' etc home usr
\ No newline at end of file
+tar czf box.apkovl.tar.gz --exclude '.DS_Store' --exclude 'README.md' etc home usr
```

Comment:
Why is README.md being excluded here?

Reply:
I'm adding README's to all dirs in repo, not sure if it makes sense to ship them into the system as well. 

The `README.md` that's being copied to the homedir is now being copied as `README.txt`, since it won't get rendered in a terminal anyway.